prime-router/src/main/resources/db/migration/V10__Update_settings_for_name_templates.sql

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
I've had a V10 languishing for weeks as part of my Auth stuff.   I guess it becomes V11....